src/http.test.ts

```diff
@@ -57,9 +65,105 @@ describe('http', () => {
         expect(body).to.equal('');
         expect(response.status).to.equal(200);
         const cookies = response.headers.get('Set-Cookie');
-        expect(cookies).to.equal(
-            'mycookie=myvalue; max-age=200000; path=/, mycookie2=myvalue; max-age=200000; path=/, mycookie3-expires=myvalue3-expires; max-age=0; path=/, mycookie4-samesite-lax=myvalue; path=/; samesite=lax, mycookie5-samesite-strict=myvalue; path=/; samesite=strict'
-        );
+        if (isOldConfig || model === 'v3') {
+            expect(cookies).to.equal(
+                'mycookie=myvalue; max-age=200000; path=/, mycookie2=myvalue; max-age=200000; path=/, mycookie3-expires=myvalue3-expires; max-age=0; path=/, mycookie4-samesite-lax=myvalue; path=/; samesite=lax, mycookie5-samesite-strict=myvalue; path=/; samesite=strict'
```

Comment:
Is there a way to make clearer the difference between these two strings? It is hard at first glance to see how they differ.